config.default.yaml

```diff
@@ -308,6 +308,27 @@ renewable:
       method: hydro_capacities  # 'hydro_capacities' to rescale country hydro production by using hydro_capacities, 'eia' to rescale by eia data, false for no rescaling
       year: 2013  # (optional) year of statistics used to rescale the runoff time series. When not provided, the weather year of the snapshots is used
     multiplier: 1.1  # multiplier applied after the normalization of the hydro production; default 1.0
+  csp:
+    cutout: cutout-2013-era5
+    resource:
+      method: csp
+      installation: SAM_solar_tower
+    capacity_per_sqkm: 4.6 # From 1.7 to 4.6 addresses issue #361
+    # Determined by comparing uncorrected area-weighted full-load hours to those
+    # published in Supplementary Data to
+    # Pietzcker, Robert Carl, et al. "Using the sun to decarbonize the power
+    # sector: The economic potential of photovoltaics and concentrating solar
+    # power." Applied Energy 135 (2014): 704-720.
+    correction_factor: 0.854337
```

Comment:
Would be great to cross-check these parameters for CSP, ideally both for solar power tower and parabolic trough. As discussed, starting points may be [NREL report ](https://www.osti.gov/biblio/1491726/) and [a fresh work ](http://dx.doi.org/10.2139/ssrn.4526901)on the global assessment of PV and CSP viability.

Totally agree on using [parameters from atlite](https://github.com/euronion/trace/blob/1b313271081a04e1fef6090ef3ec494489021035/config/config.default.yaml#L222-L245), there is also [solar_multiple_estimate](https://github.com/euronion/trace/blob/1b313271081a04e1fef6090ef3ec494489021035/config/config.default.yaml#L252C5-L252C28) which can be probably helpful.